.travis.yml

```diff
@@ -18,21 +18,24 @@ jobs:
       env: DEPENDENCIES='low'
     - php: 7.4
       env: SYMFONY_VERSION='^4.4'
+    - php: nightly
+      env: PHP8=true
 
 before_install:
   - phpenv config-rm xdebug.ini || echo "XDebug is not enabled"
 
 before_script:
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update "symfony/symfony:${SYMFONY_VERSION}"; fi;
   - if [ "$SKIP_INTEROP" != "" ]; then composer remove --dev --no-update container-interop/container-interop; fi;
-  - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
+  - if [ "$DEPENDENCIES" != "low" ]; then composer update $([ -z "$PHP8" ] || echo "--ignore-platform-reqs") ; fi;
   - if [ "$DEPENDENCIES" = "low" ]; then composer update --prefer-lowest; fi;
   - echo 'zend.exception_ignore_args=Off' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - export PATH=./bin:$PATH
 
 script:
   - ./vendor/bin/phpunit
-  - behat -fprogress --strict $([ -z "$SKIP_INTEROP" ] || echo "--tags ~@interop")
+  - behat -fprogress --strict $([ -z "$SKIP_INTEROP" ] || echo "--tags ~@interop&&~@php8")
+  - if [ "$PHP8" = true ]; then behat -fprogress --strict --tags=@php8; fi;
```

Comment:
shouldn't we rather restore the old way of automatically setting the right `@php` filters ?

Reply:
@stof is it ok to leave this for now and bring in that extension when I've had time to update it?